site/src/examples/combo-box/index.ts

```diff
@@ -12,3 +12,4 @@ export * from "./EmptyMessage";
 export * from "./Validation";
 export * from "./CustomFiltering";
 export * from "./ServerSideData";
+export * from "./ObjectValue";
```

Comment:
`ObjectValue` example needs to be added to examples.mdx for both DropdownNext & CombBoBoxNext